client/state/08types.go

```diff
@@ -56,22 +76,38 @@ func (t *taskRunnerState08) Upgrade() *state.LocalState {
 		PrestartDone: t.TaskDirBuilt,
 	}
 
+	// Don't need logmon in pre09 tasks
+	ls.Hooks["logmon"] = &state.HookState{
+		PrestartDone: true,
```

Comment:
What about if the task gets restarted (eg `template` changes)? Won't it need a logmon then? If so just letting one start just-in-case there's a restart seems ok right?